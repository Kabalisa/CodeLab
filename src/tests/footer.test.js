import Footer from '../components/footer';

describe('Footer component test', () => {
  it('should render without error', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
