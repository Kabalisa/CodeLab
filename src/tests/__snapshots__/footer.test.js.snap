// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test should render without error 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#5075D4",
      "flexDirection": "row",
      "height": "6%",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 14,
        "fontWeight": "bold",
      }
    }
  >
    Copyright Codelab 2019
  </Text>
</View>
`;
